src/Form/ImportUpdateVideoItemTrait.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\media_mpx\Form;
+
+/**
+ * Form trait for base form and utility methods.
+ *
+ * @package Drupal\media_mpx\Form
+ */
+trait ImportUpdateVideoItemTrait {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildBaseForm(array $form) {
+    if (!$video_opts = $this->loadVideoTypeOptions()) {
+      $this->messenger()->addError($this->t('There has been an unexpected problem loading the form. Reload the page.'));
```

Comment:
Weird, in that I don't see how reloading the form would fix anything. It's fine as I see this is in existing, code, just 🤔 .